themes/custom/droopler_theme/scss/components/_d_p_counters.scss

```diff
@@ -70,5 +71,10 @@ $d-p-counters-background-opacity: 0.2 !default;
         font-weight: normal;
       }
     }
+
+    .field--name-field-d-long-text {
+      color: $d-p-counters-long-text-color;
+      padding-top: 10px;
```

Comment:
Paddingi mamy przeważnie podane w `rem`, czy to pole ma też w innych paragrafach 10px paddingu?